recipe/meta.yaml

```diff
@@ -1,40 +1,47 @@
 {% set name = "cma" %}
-{% set version = "3.3.0" %}
+{% set version = "3.2.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: b748b8e03f4e7ae816157d7b9bb2fc6b1fb2fee1d5fd3399329b646bb75861ec
+  sha256: 47ac71ddd2b9a922daa601516064a585a1a3dbbae7802bca935db25e525547eb
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
-  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  skip: True # [py<36]
 
 requirements:
   host:
     - pip
-    - python >=3.6
+    - python
     - numpy
```

Comment:
Does it build without `numpy` in the `host`? I see it only in `install_requires`

```suggestion
```